project_event/models/project_task.py

```diff
@@ -286,7 +286,26 @@ def write(self, vals):
                 task.write({'responsible_id': self.responsible_id.id,
                            'partner_id': self.partner_id.id})
         elif self.activity_task_type == 'task':
-            return super(Task, self).write(vals)
+            self.ensure_one()
+            update_task = super(Task, self).write(vals)
+            if update_task and self.reservation_event_id:
+                reservation_event = self.env['calendar.event'].browse(
+                    self.reservation_event_id)
+                reservation_event.write(
+                    {
+                        'start': self.date_start,
+                        'stop': self.date_end,
+                        'name': self.name,
+                        'resource_type': self.resource_type,
+                        'room_id': self.room_id.id if self.room_id else None,
+                        'equipment_ids': [(4, self.equipment_id.id,
```

Comment:
For equipment_ids we have to reserve all the equipment inside a selected room_id. so we can have more than one equipment in the event.